README.md

```diff
@@ -149,46 +73,63 @@ A real example with a client/app id and tenant id:
+| Use managed schema | If enabled, tables, functions, and materialized views are mapped to user-friendly names. |
+| Send username header to host | With this feature enabled, Grafana will pass the logged in user's username in the `x-ms-user-id` header and in the `x-ms-client-request-id` header when sending requests to ADX. It can be useful when tracking needs to be done in ADX. |
 
-   - **Tenant Id** (Azure Active Directory -> Properties -> Directory ID)
-   - **Client Id** (Azure Active Directory -> App Registrations -> Choose your app -> Application ID)
-   - **Client Secret** ( Azure Active Directory -> App Registrations -> Choose your app -> Keys)
+### Configuring On-Behalf-Of authentication (Beta)
 
-5. Paste these three items into the fields in the Azure Data Explorer API Details section:
-   ![Azure Data Explorer API Details](https://raw.githubusercontent.com/grafana/azure-data-explorer-datasource/main/src/img/config_2_azure_data_explorer_api_details.png)
+_This feature is in Beta and subject to breaking changes_
 
-6. Click the `Save & Test` button. After a few seconds once Grafana has successfully connected then choose the default database and save again.
+For information about setting up and using the OBO flow: [on-behalf-of documentation](https://github.com/grafana/azure-data-explorer-datasource/blob/main/doc/on-behalf-of.md)
 
-### Configuring On-Behalf-Of authentication (Beta)
+## Query the data source
 
-⚠️ _This feature is in Beta and subject to breaking changes_
+Before querying the data source, select the query header options cluster, database, and format. You can create queries using the query builder, KQL, or OpenAI.
 
-For information about setting up and using the OBO flow: [on-behalf-of documentation](https://github.com/grafana/azure-data-explorer-datasource/blob/main/doc/on-behalf-of.md)
+### Query header
 
-## Writing Queries
+#### Cluster
 
-Queries are written in the new [Kusto Query Language](https://docs.microsoft.com/en-us/azure/kusto/query/).
+Select a cluster to query. If a default cluster was set in the data source settings, then it will auto-populate the cluster select. If there are no clusters to choose from, refer to [Configure the Azure Data Explorer data source](#configure-the-azure-data-explorer-data-source)
 
-Queries can be formatted as _Table_, _Time Series_, or _ADX Time Series_ data.
+#### Database
 
-### Table Queries
+Select a database to query. If a default database was set in the data source settings, it will auto-populate the database selection. 
 
-_Table_ queries are mainly used in the Table panel and row a list of columns and rows. This example query returns rows with the 6 specified columns:
+#### Format as
+
+Queries can be formatted as _Table_, _Time Series_, _Trace_, or _ADX time series_ data using the **Format as** dropdown select. 
+
+##### Table
```

Comment:
I suggest avoiding H5s whenever possible (it's hard for human eyes to distinguish them below the H4 level, and I don't think they show up in navigation on the left or right TOCs); I wonder if it would make sense to either add these queries as bullet points or in some other way, but since I'm not a subject matter expert, I'll not push the issue. 
I really like the way the Splunk data source doc is organized...but I don't know if that can be applied here. https://grafana.com/docs/plugins/grafana-splunk-datasource/latest/

Reply:
Just realized that this is a different format since it's a single README file.

Reply:
Thank you! I had a feeling I was doing something wrong with H5s. I'll reformat to use bullet points similar to Splunk's query editor bullet points.